varia/functions.php

```diff
@@ -357,6 +357,11 @@ function varia_mobile_nav_on_side_setup() {
 	add_action( 'wp_enqueue_scripts', 'varia_add_mobile_nav_on_side_scripts' );
 }
 
+/**
+ * Color palette related utilities
+ */
+require get_template_directory() . '/inc/color-utils.php';
```

Comment:
just need to make sure this doesn't break things on Atomic

Reply:
This was the Spearhead fatal on Atomic: https://github.com/Automattic/themes/pull/2865/files

Now I wonder why I didn't just just use `get_template_directory()`. hmmm...

Reply:
So I think the problem with this is that on Atomic there is no guarantee that the parent theme is installed which means that calls to functions like `change_color_luminescence` in Hever could end up causing fatal errors on Atomic.  For full context see p1606403131274200-slack-C7YPW6K40

There are two ways round this:
1. add these to the colors plugin
2. duplicate them in every theme.

I think option 1 is better but neither are great.

Reply:
Actually [on further discussion](p1610662510061000-slack-C7YPW6K40) I think this approach will be fine, but let's just make sure to test in Atomic.

Reply:
Can confirm this works on Atomic (provided that Varia is installed too, for sure)

<img width="1523" alt="Screenshot 2021-01-15 at 12 39 10" src="https://user-images.githubusercontent.com/3593343/104723047-f1b1b700-572e-11eb-92bc-745ced16fd7b.png">